lib/api.js

```diff
@@ -112,7 +112,111 @@ module.exports = {
       })
     })
       .then(handleResponse)
-  }
+  },
+  
+  getLastOracleData: function (endpoint) {
```

Comment:
I think we should also have mock,unit test for js lib

Reply:
While it's interesting, we should add another issue to improve testing, because all the API calls tests have to be implemented

Reply:
#80